pallene/driver.lua

```diff
@@ -41,7 +41,7 @@ end
 --
 -- Run AST and IR passes, up-to and including the specified pass. This is meant for unit tests.
 --
-function driver.compile_internal(filename, input, stop_after)
+function driver.compile_internal(filename, input, stop_after, opt_passes)
```

Comment:
Now might be a good time to expand the documentation commend to describe what the `stop_after` and `opt_passes` parameters are. We at least want to mention their types.